src/imagelightbox.js

```diff
@@ -542,9 +528,6 @@
                             if (!wasTouched(e.originalEvent) || options.quitOnImgClick) {
                                 return true;
                             }
-                            if (hasCssTransitionSupport) {
-                                imagePosLeft = parseInt(image.css('left'));
-                            }
```

Comment:
was that check wrong or why did you delete the line?

Reply:
stupidity

Reply:
@rejas Actually, I'm confused. That bit of code is defined for when `hasCssTransitionSupport` is truthy, but the `imagePosLeft` is used for when `hasCssTransitionSupport` is falsey. What gives?

Reply:
It's like this already in the original code. I am wondering how it would look like if we change this line....